cli/src/migrateMongo/index.js

```diff
@@ -80,8 +81,32 @@ const getOutstandingMigrations = async ({ down, up, migrations }) => {
   return outstandingMigrations;
 };
 
+export const fixOrderFn = async ({ migrations }) => {
+  const dbMigrations = await migration.find({}).sort({ order: 1 });
+  let toSave = [];
+
+  let inc = 1;
+  for (const migratio of migrations.keySeq().toJS()) {
```

Comment:
I think missing letters off like in `migratio` makes the code here harder to read. Can you try to name these variables a little clearer please.